docs/book/persistence.md

```diff
@@ -71,19 +71,12 @@ session:
 The first approach is problematic when using mezzio-session in an async
 environment such as [Swoole](https://swoole.co.uk) or
 [ReactPHP](https://reactphp.org), as the same persistence instance may be used
-by several simultaneous requests. As such, version 1.1.0 introduces a new
-interface for `Mezzio\Session\SessionInterface` implementations to use:
-`Mezzio\Session\SessionIdentifierAwareInterface`. This interface
-defines a single method, `getId()`; implementations can thus store the
+by several simultaneous requests. `Mezzio\Session\SessionInterface` defines a new
+`getId` method, implementations can thus store the
 identifier internally, and, when it comes time to store the session data,
 persistence implementations can query that method in order to retrieve the
 session identifier.
 
-Considering that persistence implementations also _create_ the session instance,
-we recommend that implementations only create instances that implement the
-`SessionIdentifierAwareInterface` going forward in order to make themselves
-async compatible.
-
 ## Persistent sessions
```

Comment:
All the changes to the docs need to be reverted.

First of all, we need a patch to `1.x` to move the docs from the root `/docs/` folder to `/docs/v1`, ensuring we set up redirects for `/some-file.md` to `/v1/some-file.md`

Example: https://github.com/laminas/laminas-validator/pull/244

This needs to be released and merged up into `2.x`

Then, we need a patch to `2.x` that duplicates the `v1` docs tree to a `v2` folder:

Example: https://github.com/laminas/laminas-validator/pull/248.

At this point, we'll need to rebase this patch, and then make the required changes to the docs in the v2 tree.

Make sense?

Reply:
Sure, I forgot to make the v2 folder - I'll fix it.

Reply:
See #63 for the `v1` documentation modifications.